From 2c6292a060a243aca35af2463295a51a8853c3c6 Mon Sep 17 00:00:00 2001
From: Adrian Taylor <adetaylor@chromium.org>
Date: Tue, 15 Feb 2022 12:01:06 -0800
Subject: [PATCH] Unpin proc-macro-error dependencies.

proc-macro-error pins serde_derive in its dev-dependencies,
and also toml. This is (at least partly) due to a need for
compatibility with older rustc versions, which doesn't
matter to us. This pinning causes problems in our epoch-based
world where we can only have one version of a crate within
a specific semver epoch.

Change-Id: Ie98958e18a97269f3c3dad2621038f32d3e755a3
---
 third_party/rust/proc_macro_error/v1/crate/Cargo.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/third_party/rust/proc_macro_error/v1/crate/Cargo.toml b/third_party/rust/proc_macro_error/v1/crate/Cargo.toml
index 869585ffc247b..6705b2453a3c3 100644
--- a/third_party/rust/proc_macro_error/v1/crate/Cargo.toml
+++ b/third_party/rust/proc_macro_error/v1/crate/Cargo.toml
@@ -38,10 +38,10 @@ version = "1"
 optional = true
 default-features = false
 [dev-dependencies.serde_derive]
-version = "=1.0.107"
+version = "1.0.107"
 
 [dev-dependencies.toml]
-version = "=0.5.2"
+version = "0.5.2"
 
 [dev-dependencies.trybuild]
 version = "1.0.19"
-- 
2.35.1.265.g69c8d7142f-goog

